modulesets-stable/gtk-osx-bootstrap.modules

```diff
@@ -80,7 +80,7 @@
   <!-- Cups is needed *only* for Tiger, and skipped otherwise -->
   <autotools id='cups' autogen-sh='configure' skip-autogen='never'
 	     autogenargs='DSOFLAGS="$LDFLAGS"'>
-    <branch module="cups/1.2.12/cups-1.2.12-source.tar.bz2"
+    <branch module="1.2.12/cups-1.2.12-source.tar.gz"
```

Comment:
I prefer bz2 archives over gz because they're smaller. Both are available at cups.org, why do you want to change?